rat/variables.py

```diff
@@ -70,7 +70,7 @@ def compute_shifted_df(self, df, shifts):
 
         shifted_columns = []
         for column, shift in zip(shift_columns, shift_values):
-            shifted_column = grouped_df[column].shift(shift).reset_index(drop=True)
+            shifted_column = grouped_df[column].shift(int(shift)).reset_index(drop=True)
```

Comment:
It looks like the shift values are being passed to the compiler as strings. If you remove this integer cast things fail (and it should work without this cast)

Reply:
Changed so parser returns int